origin-trials/downstream-trials.json

```diff
@@ -9,16 +9,10 @@
             "flag": "msAcquisitionInfo"
         },{
             "label": "Digital Goods API",
-            "expiration": "2023-12-31",
-            "repo": "MicrosoftEdge/MSEdgeExplainers",
-            "explainer": "#"
-        },{
-            "label": "SystemEntropy in PerformanceNavigationTiming",

```

Comment:
This is likely to get extended shortly. Can we keep this here for now?

Reply:
SystemEntropy in PerformanceNavigationTiming has been un-expired. Did you want to keep Web Haptics too, or should I leave it as-is?

Reply:
I'm not sure who is using that one - @SteveBeckerMSFT what do you want to do with WebHaptics?

Reply:
I removed web haptics to unblock this change. @mwjacksonmsft does this look okay now that it's only digital goods?

Reply:
I think we can leave WebHaptics expired.  We are running a private origin trial only.  It is likely that we will delete this API from Edge during 2024.